src/utils/nodeTemplates.ts

```diff
@@ -81,3 +82,10 @@ export function createUint256TypeName(ast: AST): ElementaryTypeName {
   ast.setContextRecursive(typeName);
   return typeName;
 }
+
+export function createParameterList(
+  params: Iterable<VariableDeclaration>,
+  ast: AST,
+): ParameterList {
+  return new ParameterList(ast.reserveId(), '', 'ParameterList', params);
```

Comment:
Very much like to see more functions getting added here, I'm hoping over time a large number of the raw constructors used will get replaced with such functions. I do think it would be worth calling `ast.setContextRecursive` on the generated parameterList though